### ISSUE TITLE
Find, next = crash

### ISSUE THREAD

**Issue by clessley** [notepadqq/notepadqq#142]
When performing a Find, dialog box opens, search as type find first, nearest result. Hitting enter key or next to find next result immediately crashes notepadqq without error, just completely dies/disappears on enter or click.

On Ubuntu 15.04 MATE. Will provide error log if directed to location of log, if exists.

**Comment by darealshinji** [comment]
Open Notepadqq in a terminal, maybe there will be useful information printed on there.

**Comment by gripfastartworks** [comment]
Will do. I'll report back on git tonight.
project coming along nicely BTW!

-------- Original message --------

**Comment by gripfastartworks** [comment]
/usr/lib/notepadqq/notepadqq-bin: symbol lookup error: /usr/lib/notepadqq/notepadqq-bin: undefined symbol: _ZN9QtPrivate20QContainerImplHelper3midEiPiS1_

**Comment by darealshinji** [comment]
What are the DT_NEEDED entries of your notepadqq binary (run `readelf -d /usr/lib/notepadqq/notepadqq-bin | grep NEEDED`)?

I'm using a version that I've compiled myself (Ubuntu 15.04) and never had such a problem.

```
djcj: ~ $ LANG=C readelf -d /usr/lib/notepadqq/notepadqq-bin | grep NEEDED                                                                                                                                        
 0x0000000000000001 (NEEDED)             Shared library: [libQt5WebKitWidgets.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libQt5PrintSupport.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libQt5Widgets.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libQt5WebKit.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libQt5Gui.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libQt5Network.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libQt5Core.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
```

**Comment by gripfastartworks** [comment]
Here are the entries:

readelf -d /usr/lib/notepadqq/notepadqq-bin | grep NEEDED
 0x00000001 (NEEDED)                     Shared library: [libQt5WebKitWidgets.so.5]
 0x00000001 (NEEDED)                     Shared library: [libQt5PrintSupport.so.5]
 0x00000001 (NEEDED)                     Shared library: [libQt5Widgets.so.5]
 0x00000001 (NEEDED)                     Shared library: [libQt5WebKit.so.5]
 0x00000001 (NEEDED)                     Shared library: [libQt5Gui.so.5]
 0x00000001 (NEEDED)                     Shared library: [libQt5Network.so.5]
 0x00000001 (NEEDED)                     Shared library: [libQt5Core.so.5]
 0x00000001 (NEEDED)                     Shared library: [libstdc++.so.6]
 0x00000001 (NEEDED)                     Shared library: [libgcc_s.so.1]
 0x00000001 (NEEDED)                     Shared library: [libc.so.6]

**Comment by danieleds** [comment]
Still having the problem? Are you using the version from the ppa?

**Comment by gripfastartworks** [comment]
removed previous install, apt-get remove followed apt-get autoremove. then reinstalled from ubuntu ppa;

sudo add-apt-repository ppa:notepadqq-team/notepadqq
sudo apt-get update
sudo apt-get install notepadqq

new install, find next still crashes notepadqq, wondering, are there any specific unity dependencies that I might be missing?

**Comment by danieleds** [comment]
No, there are not any unity-related dependencies.
What version of Qt is Notepadqq picking up? `? -> About Qt`

**Comment by gripfastartworks** [comment]
![about](https://cloud.githubusercontent.com/assets/12699698/9723386/e7c6696e-5574-11e5-8f12-07ceebf0fd1f.png)

**Comment by clessley** [comment]
I ended up deleting the entire QT folder within /home/username/ directory and reinstalling the open source x86 run file from the all downloads page at the QT project at: http://download.qt.io/official_releases/online_installers/qt-unified-linux-x86-online.run. This loaded QT 5.5.1 along with the Creator and Maintainence tool (which I never had before when I originally reported the bug on 9/7/2015).

Fixed my find > find next crash. Everything seems ok now! On Ubuntu 16.04 beta MATE 1.12.1.

![screenshot at 2016-04-04 20 31 20](https://cloud.githubusercontent.com/assets/12699698/14271000/aa54f81e-faa8-11e5-8854-9605959becf3.png)
